examples/hybrid/driver.jl

```diff
@@ -69,10 +77,27 @@ additional_cache(Y, params, dt; use_tempest_mode = false) = merge(
         rrtmgp_model_cache(Y, params; idealized_h2o),
     isnothing(turbconv) ? NamedTuple() :
         vertical_diffusion_boundary_layer_cache(Y),
+    (;
+        tendency_knobs = (;
+            hs_forcing = forcing == "held_suarez",
+            const_diff = turbconv == "const_diff",
+            microphy_0M = microphy == "0M",
+            rad_flux = !isnothing(rad),
+            hyperdiff,
+        ),
+    ),
 )
 
-additional_tendency!(Yₜ, Y, p, t) = nothing
-postprocessing(sol, output_dir) = nothing
+additional_tendency!(Yₜ, Y, p, t) = begin
+    (; rad_flux, const_diff, hs_forcing) = p.tendency_knobs
+    (; microphy_0M, hyperdiff) = p.tendency_knobs
+    hyperdiff && hyperdiffusion_tendency!(Yₜ, Y, p, t)
+    sponge && rayleigh_sponge_tendency!(Yₜ, Y, p, t)
+    hs_forcing && held_suarez_tendency!(Yₜ, Y, p, t)
+    const_diff && vertical_diffusion_boundary_layer_tendency!(Yₜ, Y, p, t)
```

Comment:
`const_diff` is not a proper name for this particular function. `vertical_diffusion_boundary_layer_tendency!` is following the paper where diffusivity is not constant.

Reply:
What do you recommend we call it?

Reply:
maybe `vert_diff_BL`

Reply:
Actually, `vert_diff` was proposed above, I'll stick with that for now, I think we can further improve names down the road, though.